src/exportNameRule.ts

```diff
@@ -46,26 +46,43 @@ export class Rule extends Lint.Rules.AbstractRule {
     }
 }
 
+function isExportedDeclaration(element: ts.Statement): boolean {
```

Comment:
Just a suggestion: you may wish to file issues on https://github.com/ajafff/tsutils to add these there, as they seem useful. Not a blocking issue for this pull request at all.

Reply:
You got it: https://github.com/ajafff/tsutils/issues/68 - feel free to comment there if it's not clear or not quite what you expected.